sycl/test/basic_tests/image_accessor_readwrite.cpp

```diff
@@ -0,0 +1,413 @@
+// RUN: clang++ -fsycl %s -o %t.out -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out
+//==----- image_accessor_write_api.cpp - image_accessor write API test-----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <cassert>
+#include <iomanip>
+#include <iostream>
+
+namespace s = cl::sycl;
+
+using namespace s;
```

Comment:
Why do you "using" namespace s, not sure you are using it.

Reply:
I'm using it at some lcoations. I will make it consistent.

Reply:
fixed.

Reply:
I suggest removing this line.
I remember there were some name conflicts between the symbols from `cl::sycl` name space and some other libraries.
It's better to use fully qualified names in the test.
Please, use the alias defined at the line 21.